float32 yaw
float32 pitch
float32 roll
float32 motor11
float32 motor12
float32 motor13
float32 motor14
float32 motor15
float32 motor16
float32 motor17
float32 motor18
